WordPress/Classes/ViewRelated/Comments/CommentDetailInfoViewController.swift

```diff
@@ -52,7 +53,10 @@ final class CommentDetailInfoViewController: UIViewController {
 
     private func setPreferredContentSize() {
         tableView.layoutIfNeeded()
-        preferredContentSize = tableView.contentSize
+
+        if UIDevice.isPad() {
+            preferredContentSize = CGSize(width: 320, height: tableView.contentSize.height)
+        }
```

Comment:
I wonder if you can delete this setting preferredContentSize code if setting `tableView.scrollable = false`.

Reply:
Or other ways to make `UITableView` display its full content.

I'm not sure using `UITableView` is a good choice here... I presume UIKit would take autolayout into account when displaying a popover. We probably don't need to set `preferredContentSize` if using a `UIStackView` instead.

Reply:
This was the existing implementation. I'm not planning to rework it in the scope of this PR.